src/renderer/api/catalog-entity-registry.ts

```diff
@@ -49,23 +46,27 @@ export class CatalogEntityRegistry {
     return this._activeEntity;
   }
 
-  get items() {
-    return this._items;
+  @computed get items() {
```

Comment:
I think we should have tests that confirm filtering (now that it starts to be even more complex).